sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,4 @@
+-- Ironhand Guardion ScriptName
+SET @ENTRY := 8982;
+UPDATE `creature_template` SET `AIName`='', ScriptName='npc_ironhand_guardian' WHERE `entry`=@ENTRY;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=@ENTRY;
```

Comment:
sourcetype is always required here

Reply:
he is deleting the whole SAI of mob here, no need of sourcetype.

Reply:
even if there is no data for other types, always use sourcetype when deleting rows